imgui-sys/third-party/cimgui_impl.h

```diff
@@ -1,38 +0,0 @@
-typedef struct SDL_Window SDL_Window;
```

Comment:
Unsure what the story here is but we don't provide the functions these correspond to in the static lib our -sys crate builds, nor do we expose them as rust functions, so I think it was just a mistake or cimgui quirk.